src/descriptors/power_system_structs.json

```diff
@@ -139,24 +136,26 @@
         "data_type": "Float64"
       },
       {
-        "name": "activepower",
-        "null_value": "0.0",
-        "data_type": "Float64"
+        "name": "primemover",
+        "comment": "PrimeMover Technology according to EIA 923",
+        "null_value": "nothing",
+        "data_type": "Union{Nothing,PrimeMovers}"
```

Comment:
The PrimeMovers enum defines an OTHER value, so an instance could be set to either nothing or OTHER.  That could add complexity to higher-level code.  Could we allow only one option and delete the other?

Reply:
Done